### PR TITLE
Add Discord webhook for Pretalx updates

### PR DESCRIPTION
**Pull request by mattcen**
According to [Discord's webhook docs](https://support.discord.com/hc/en-us/articles/228383668-Intro-to-Webhooks):

1. Open your Server Settings and head into the Integrations tab
2. Click the "Create Webhook" button to create a new webhook
3. Get the webhook URL for the server/channel you want to receive messages
4. Ensure to append `/github` to the end of the URL

Then on the GitHub side, we're not going to use the classic Web hook integration; we're doing this from an Action.

1. Make the changes listed in this commit (using a [GitHub Webhook action](https://github.com/marketplace/actions/workflow-webhook-action))
2. And set the `WEBHOOK_URL` secret fot the repo to the aforementioned URL

### CODE REVIEW

**Comment by yaakov-h** [comment]
well that looks surprisingly simple

**Comment by Mause** [comment]
Is this going to just dump a json message into the discord channel at a regular interval?

**Comment by yaakov-h** [comment]
That's a good point, we only really want to be alerted if `EndBug/add-and-commit@v4` made any changes.

**Comment by Mause** [comment]
That is an output that the action provides yeah: https://github.com/EndBug/add-and-commit#outputs

But it's still going to just be a json payload right?

**Comment by mattcen** [comment]
Yes, you're right; it needs to be edited to only run if add-and-commit makes changes; I'll dig into this.

As for what'll end up in Discord, the docs weren't *super* clear as to how clever it was insofar as parsing the content of the request. so I don't know exactly what appears in the channel.

**Comment by mattcen** [comment]
My rationale though was that even if we can just get it to show a commit hash we can click on, given that these edits only happen a couple of times a day (looking at recent commits), that's *probably* fine.

**Comment by Mause** [comment]
I'm actually not sure it will do anything at all if it's not matching the schema that discord is expecting? 

Maybe something like https://github.com/marketplace/actions/notify-discord instead

**Comment by mattcen** [comment]
OK, updated the code to use the Discord webhook action and only run if a commit happened. Allegedly. How does it look?

File: .github/workflows/update_schedule.yml [hunk 1]
```diff
@@ -31,3 +32,11 @@ jobs:
           message: "Auto-update of schedule"
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Notify discord
+        if: ${{ steps.add-and-commit.outputs.committed }}
```
Comment (Mause) [inline .github/workflows/update_schedule.yml:36]:
```suggestion
        if: steps.add-and-commit.outputs.committed == 'true'
```

**Review by Mause — CHANGES_REQUESTED** [review]

**Review by Mause — APPROVED** [review]

**Review by yaakov-h — APPROVED** [review]
LPTM (looks plausible to me) :shipit:

**Pull request merged**